helpers/class.TestSession.php

```diff
@@ -150,26 +146,6 @@ protected function getResultServer()
         return $this->resultServer;
     }
 
-    /**
-     * Set the ResultTransmitter object to be used to transmit test results.
-     *
-     * @param taoQtiCommon_helpers_ResultTransmitter $resultTransmitter
-     */
-    protected function setResultTransmitter(taoQtiCommon_helpers_ResultTransmitter $resultTransmitter)
-    {
-        $this->resultTransmitter = $resultTransmitter;
-    }
-
-    /**
-     * Get the ResultTransmitter object to be used to transmit test results.
-     *
-     * @return taoQtiCommon_helpers_ResultTransmitter
-     */
-    protected function getResultTransmitter()
```

Comment:
Removing these two methods is technically a breaking change.
Although, I don't see any other extension overriding or using them, I strongly suggest bumping a major version of the extension, as well as explicitly mentioning what exactly was a breaking change in the future release notes.

Reply:
my bad, miss it. will add breaking changes